ds-card-reader/res/devices/device-list.toml

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 [[DeviceList]]
-  Name = 'card-reader'
+  Name = 'ds-card-reader'
```

Comment:
The device name should not be the same as the service name. Please revert this chnage.